tests-ng/plugins/booted.py

```diff
@@ -1,24 +1,29 @@
 import pytest
+from typing import Any, List
 
 system_booted = False
 
-def is_system_booted():
+
+def is_system_booted() -> bool:
     return system_booted
 
-def pytest_addoption(parser):
+
+def pytest_addoption(parser: pytest.Parser):
```

Comment:
If nothing is returned, usually `-> None` [1] is defined in these cases.

[1] https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html

Reply:
compare to @ccwienk 's comment, it seems like there is no consensus on that. I don't have a strong opinion on that.